source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -118,10 +118,10 @@
             if (this.evaluation.readyForDevolution) {
                 return this.userView == 3 && (calification.calificationColumn.owner == 3 || calification.calificationColumn.owner == 0);
             } else {
-                return (((this.userLogged == calification.calificationColumn.evaluatorEmployee)
-                || (this.userView == 3 && calification.calificationColumn.evaluatorEmployee == "_company"))
-                && !calification.calificationColumn.finished)
-            }
+                return (!calification.calificationColumn.finished
+                   && ((this.userLogged == calification.calificationColumn.evaluatorEmployee && this.isCompanyEvaluationDone)
+                       || (this.userView == 3 && calification.calificationColumn.evaluatorEmployee == "_company")
+                    ))}
```

Comment:
Do you think that it is possible to simplify `isValueEditable` function? it is not easy to read.